airbyte-integrations/connectors/source-googleanalytics-singer/airbyte_protocol

```diff
@@ -0,0 +1 @@
+../../bases/base-python/airbyte_protocol
```

Comment:
what is the purpose of this wrapper file? does this facilitate imports or something?

Reply:
it's a symlink, and it makes the IDE work nicer in some cases.